### DISCUSSION TITLE
Can I add my own BIOS images?

### DISCUSSION THREAD

**Discussion by robman501a** [86Box/86Box#3128] [Q&A] [answered]
Hi, is it possible to add my own BIOS images in to 86Box?
For example, I used to have the ASUS P2B-DS and that had an Adaptec SCSI controller with the normal IDE controller. Can I just jump on to the ASUS support page, download the BIOS, make a folder in 86Box\roms\p2bds and place the image in there?

**Comment by jriwanek** [comment, marked as answer]
No, you'd have to add support for it, and any of it's additional hardware to 86Box, that means modifying code.
